src/librustdoc/html/render/print_item.rs

```diff
@@ -1867,7 +1867,7 @@ fn item_proc_macro(cx: &Context<'_>, it: &clean::Item, m: &clean::ProcMacro) ->
                     }
                 }
             }
-            Ok(())
+            Ok::<(), fmt::Error>(())
```

Comment:
A `fmt::Result::Ok` might be nicer here, maybe? Just a random insignificant suggestion 😅

Reply:
It's actually a pretty good idea, thanks!